content/home-2020-oct/card_5/en-US.md

```diff
@@ -0,0 +1,11 @@
+---
+# card 5
+title: "Get the latest Firefox"
+tag_label: "Firefox"
+desc: "Choose the only non-profit-backed, people-first browser."
+link_url: "https://www.mozilla.org/firefox/"
```

Comment:
IMHO this should go to /new but I'll leave it up to you to talk to MJ or not or maybe you already have.

Reply:
Updated to https://www.mozilla.org/en-US/firefox/new/!